src/cart/crt_rpc.c

```diff
@@ -29,6 +29,17 @@
 
 #define CRT_CTL_MAX_LOG_MSG_SIZE 256
 
+int
+crt_req_set_retry(struct crt_rpc_priv* rpc_priv) {
+        return (atomic_fetch_or_relaxed(&rpc_priv->crp_rpc_retry, CRT_RPC_RETRY) & ~CRT_RPC_RETRY);
+}
+
+int
+crt_req_reset_retry(struct crt_rpc_priv* rpc_priv)
+{
+        return (atomic_fetch_and_relaxed(&rpc_priv->crp_rpc_retry, ~CRT_RPC_RETRY));
```

Comment:
(style)  line over 80 characters
(style)  code indent should use tabs where possible
(style)  return is not a function, parentheses are not required